client/src/components/coursePage.jsx

```diff
@@ -0,0 +1,38 @@
+import React, { useState } from 'react';
+import CourseCard from './courseCard';
+import '../stylings/coursePage.css';
+
+const courses = [
```

Comment:
Similar to above comments. Keep temporary changes like temp data local and not commit them.